gcloud/streaming/transfer.py

```diff
@@ -190,14 +301,26 @@ def __repr__(self):
                 self.progress, self.total_size, self.url)
 
     def configure_request(self, http_request, url_builder):
+        """Update http_request/url_builder with download-appropriate values.
+
+        :type http_request: :class:`gcloud.streaming.http_wrapper.Request`
+        :param http_request: the request to be updated
+
+        :type url_builder: instance with settable 'query_params' attribute.
+        :param url_builder: transfer policy object to be updated
+        """
         url_builder.query_params['alt'] = 'media'
-        # TODO(craigcitro): We need to send range requests because by
-        # default httplib2 stores entire reponses in memory. Override
-        # httplib2's download method (as gsutil does) so that this is not
-        # necessary.
+        # We send range requests because by default httplib2 stores entire
+        # reponses in memory. Consider overriding # httplib2's download method
```

Comment:
You have a `#` in the middle of a line.

Also the `Consider` should either be deleted or filed as a bug.